lang_python/analyze/Python_to_generic.ml

```diff
@@ -280,6 +280,8 @@ let rec expr (x: expr) =
       G.Await (t, v1)
   | Repr v1 -> let (_, v1, _) = bracket expr v1 in
       G.OtherExpr (G.OE_Repr, [G.E v1])
+  | NamedExpr ((v, t, e)) ->
+      G.Assign(expr v, t, expr e)
```

Comment:
note that OCaml order of evaluation is from right to left ... yes it's weird, which means
expr e will be executed before expr v. In this case it should not matter, but sometimes when those functions
have side effects it can matter. Just FYI.